xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1073,7 +1073,7 @@ _io_stats_write_latency_sample(xlator_t *this, ios_sample_t *sample,
     conf = this->private;
 
     epoch_time = (sample->timestamp).tv_sec +
-                 ((sample->timestamp).tv_usec / 1000000.0);
+                 ((sample->timestamp).tv_nsec / 1000000000.0);
```

Comment:
nit: I find it easier to read with 1e9 instead.

Reply:
Sure! Changed to '1e9'